src/faebryk/libs/picker/picker.py

```diff
@@ -268,9 +269,13 @@ def check_missing_picks(module: Module):
         )
         # no parent with picker
         and not try_or(
-            lambda: m.get_parent_with_trait(F.is_pickable),
-            default=False,
-            catch=KeyErrorNotFound,
+            lambda: try_or(
```

Comment:
Love the concept of `try_or`, but found it unwieldy, especially to read in practice.

Perhaps they'd be clearer w/ a partial? I've played with making a `try_or` with a signature something like `partial`:
```python
def try_or[T](func: Callable[[*A, **K], T], *A, excepts: type[Exception], default: T = None):
```
Might be clearer using partials rather than lambdas?

Ideally though, if nesting is going beyond a single walrus-stmt, I'd suggest we make a true function rather than a lambda for the key/filter/matcher. These deep, deep nested lambdas are also pretty hard for at least me, and I expect EEs and others getting used to the core to follow.